sdks/python/apache_beam/examples/complete/game/game_stats_it_test.py

```diff
@@ -52,7 +52,7 @@
 class GameStatsIT(unittest.TestCase):
 
   # Input events containing user, team, score, processing time, window start.
-  INPUT_EVENT = 'user1,teamA,10,%d,2015-11-02 09:09:28.224'
+  INPUT_EVENT = b'user1,teamA,10,%d,2015-11-02 09:09:28.224'
```

Comment:
I am not sure if this should be bytes. Does self.pub_client.publish require bytes? If so, we should encode before passing data to that method, 

To quote https://docs.python.org/3/howto/pyporting.html ,  `Decode binary data to text as soon as possible, encode text as binary data as late as possible`